### PR TITLE
Update GVFS, Kerberos5, Apache Ant and OpenJDK Flathub Extension

### PR DESCRIPTION
**Pull request by EliasOfWaffle**
Update GVFS to the newest release 1.50.4 
Apache ant to 1.10.9.
Kerberos 5 to 20.1
Use new OpenJDK 17 LTS version instead 11

### CODE REVIEW

File: org.libreoffice.LibreOffice.json [hunk 1]
```diff
@@ -64,8 +64,8 @@
                 },
                 {
                     "type": "archive",
-                    "url": "https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.tar.xz",
-                    "sha256": "cebb705dbbe26a41d359b8be08ec066caba4e8686670070ce44bbf2b57ae113f",
+                    "url": "apache-ant-1.10.9-bin.tar.xz",
+                    "sha256": "42f12ea7dc854b69aac0bd05c5767252dddd297888503e69b8aae0a1dc791cf5",
```
Comment (stbergmann) [inline org.libreoffice.LibreOffice.json:68]: Why would you want to do all these changes here?

The use of external project versions here is exactly the same as in the upstream LO version. The list here and the upstream list at <https://git.libreoffice.org/core/+/refs/tags/libreoffice-7.5.1.2/download.lst> are kept in sync with a tool from <https://git.libreoffice.org/core/+/c5f620f9f6abae0213fdda17fae10f281b7bdf9f%5E!/> "add generate-flatpak-manifest.sh to generate FlatPak manifest".

**Review by stbergmann — CHANGES_REQUESTED** [review]

File: org.libreoffice.LibreOffice.json (hunk 1 above)
Comment (EliasOfWaffle) [reply org.libreoffice.LibreOffice.json:68]: oh thanks, sorry for my mistake, i not know that have a list for use specific version

Comment (EliasOfWaffle) [reply org.libreoffice.LibreOffice.json:68]: > Why would you want to change this?
> 
> The use of external project versions here is exactly the same as in the upstream LO version. The list here and the upstream list at https://git.libreoffice.org/core/+/refs/tags/libreoffice-7.5.1.2/download.lst are kept in sync with a tool from https://git.libreoffice.org/core/+/c5f620f9f6abae0213fdda17fae10f281b7bdf9f%5E!/ "add generate-flatpak-manifest.sh to generate FlatPak manifest".

sorry but the ant is not in this list

Comment (EliasOfWaffle) [reply org.libreoffice.LibreOffice.json:68]: ant look's to be external jar archive release, is the same in build in my notebook, download a apache ant jar and define the location in path

Comment (EliasOfWaffle) [reply org.libreoffice.LibreOffice.json:68]: Ant looks to be a independent from Libreoffice lists, apache ant is a jar file, and ant is a build-system for Java code, it's is similar to makefile, can be good use a newest release, it's not break compatiblity.
GVFS have a newest release too, can be good to use this, and Java 17 is not more newest then Java 19 but can be good to use a newest LTS release

Comment (stbergmann) [reply org.libreoffice.LibreOffice.json]: Yeah, sorry for the confusion on my part.  Ant is indeed a prerequisite that we do not bundle as an external source with LO upstream.
So you could either add your update of Ant back into this PR, or let me know whether you're fine with the current state of the PR (without updated Ant) in which case I'd go ahead and push it.
In any case, sorry for the back and forth, and thanks for the patch!

File: org.libreoffice.LibreOffice.json [hunk 2]
```diff
@@ -64,8 +64,8 @@
                 },
                 {
                     "type": "archive",
-                    "url": "https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.tar.xz",
-                    "sha256": "cebb705dbbe26a41d359b8be08ec066caba4e8686670070ce44bbf2b57ae113f",
+                    "url": "https://dlcdn.apache.org//ant/binaries/apache-ant-1.10.13-bin.zip",
```
Comment (stbergmann) [inline org.libreoffice.LibreOffice.json:67]: * Is the change from `archive.apache.org` to `dlcdn.apache.org` intentional or by accident?  (If intentional, a rationale in the commit message would probably be helpful.)
* The double `//` in the middle of the URL should be a single `/`.
* Is the change from `.tar.xz` to `.zip` intentional or by accident?  (If intentional, a rationale in the commit message would probably be helpful.)

**Review by stbergmann — CHANGES_REQUESTED** [review]

**Pull request merged**